scripts/release-candidate.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+# date from git tag
```

Comment:
It would be nice if you can document by example: 
* an example of a git tag that function expects.
* an example of a date extracted from it.

Also, wouldn't it be better to give argument a name instead of relying on `$1` ?

Reply:
I just copied the code from make_release.sh. I will add an example anyway.